Correct one line calling powershell (#16425)
dotnet/docs: docs/core/tools/dotnet-install-script.md

  Windows:

  ```powershell

>   #Run an seperate powershell process because the script calls exit and so will end the current powershell session.

Reviewer: ```suggestion
  # Run a separate PowerShell process because the script calls exit, so it will end the current PowerShell session.
```

Author: Thanks for pointing out my poor grammar.

Reviewer: No worries! Make sure you click the button Commit suggestion to get them incorporated.
